gdpr/impl.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 
 	"github.com/prebid/go-gdpr/api"
-	tcf1constants "github.com/prebid/go-gdpr/consentconstants"
-	consentconstants "github.com/prebid/go-gdpr/consentconstants/tcf2"
+	"github.com/prebid/go-gdpr/consentconstants"
+	tcf2ConsentConstants "github.com/prebid/go-gdpr/consentconstants/tcf2"
```

Comment:
Is there a reason for changing this name to tcf2 specific?

Reply:
I was trying to maintain the differentiation between the `consentconstants` and `consentconstants/tcf2` packages while removing any reference to tcf1 so I renamed both of these.